Tools/ide/vscode/template/tasks.json

```diff
@@ -24,14 +24,14 @@
             "label": "Full rebuild (no debugging0",
             "detail": "Rebuild application and all Components",
             "type": "shell",
-            "command": "make clean components-clean SMING_ARCH=${command:cpptools.activeConfigName} && make -j ENABLE_GDB=",
+            "command": "make clean components-clean SMING_ARCH=${command:cpptools.activeConfigName}; make -j ENABLE_GDB=",
```

Comment:
VSCode generator is used also for Linux, right? It must be distinguished for this OS. Probably using some variable.

Reply:
Good point. Maybe there's a cleaner way to list multiple commands...

Reply:
Looks like there can be an override for Windows specific:
https://code.visualstudio.com/docs/editor/tasks#_custom-tasks

Reply:
Btw. I just found the `;` works only for Poweshell :-). 
For Cmd.exe there should be && :-)). What a nice mess :-).

Reply:
I think the proper way might be a compound task:
https://code.visualstudio.com/docs/editor/tasks#_compound-tasks

Reply:
Compound tasks: exactly what I was looking for. Thank you!

Reply:
Excellent!